posthog/hogql_queries/ai/traces_query_runner.py

```diff
@@ -215,8 +215,8 @@ def _get_event_query(self) -> ast.SelectQuery:
                         argMin(person.created_at, timestamp), argMin(person.properties, timestamp)
                     ) as first_person,
                     round(toFloat(sum(properties.$ai_latency)), 2) as total_latency,
-                    sum(properties.$ai_input_tokens) as input_tokens,
-                    sum(properties.$ai_output_tokens) as output_tokens,
+                    sum(toFloat(properties.$ai_input_tokens)) as input_tokens,
```

Comment:
why not toDecimal32 or equivalent?

Reply:
No specific reason, just that we already mapped cost and latency to float. So for calculations using all floats felt consistent.  Happy to update